content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -356,7 +363,7 @@ Prisma Client mitigates the risk of SQL injection in the following ways:
 
 - `$executeRaw` can only run **one** query at a time. You cannot append a second query - for example, adding `DROP bobby_tables` to the end of an `ALTER`.
 
-If you cannot use tagged templates, you can instead use [`$queryRawUnsafe`](#query-raw-unsafe) or [`$executeRawUnsafe`](#execute-raw-unsafe) but **be aware that your code may be vulnerable to SQL injection**.
+If you cannot use tagged templates, you can instead use [`$queryRawUnsafe`](#queryrawunsafe) or [`$executeRawUnsafe`](#executerawunsafe) but **be aware that your code may be vulnerable to SQL injection**.
```

Comment:
old anchors were added manually again above - not necessary?

Reply:
See other comments